devices/IWearRemapper/src/IWearRemapper.cpp

```diff
@@ -104,7 +165,16 @@ bool IWearRemapper::open(yarp::os::Searchable& config)
         pImpl->waitForAttachAll = config.find("waitForAttachAll").asBool();
     }
 
-
```

Comment:
The parameter `allowDynamicData` is set to mandatory, we should remember to spread around this modification in the configuration files (in this PR we could address the changes for what concern this repo).

We can also think of making the parameter as option (and default value `true` to keep the previous behaviour)

Reply:
Actually it is not mandatory, you just get a INFO log message when it's missing or not of type bool

Reply:
Sorry, you're right